src/translation/english.c

```diff
@@ -64,6 +64,7 @@ static translation_string all_strings[] = {
     {TR_HOTKEY_RESIZE_TO_1024, "Resize window to 1024x768"},
     {TR_HOTKEY_SAVE_SCREENSHOT, "Save screenshot"},
     {TR_HOTKEY_SAVE_CITY_SCREENSHOT, "Save full city screenshot"},
+    {TR_HOTKEY_BUILD_CLONE, "Clone selected building"},
```

Comment:
I think a better description would be "Clone building under cursor". "Selected" might confuse users.